src/Pages/Events/EventCard.js

```diff
@@ -16,38 +15,35 @@ function EventCard(props) {
   } = props;
 
   return (
-    <Container
-      className='event-container'
-      onClick={() => {
-        handleClick();
-      }}
-    >
-      <Row className='card-row'>
-        <Col>
+    <div>
+      <button
+        className='event-card-button'
+        onClick={() => {
+          handleClick();
+        }}
+      >
+        <tbody>
+          <td>
+            <img id='event-image'
+              className='event-image'
+              src={imageURL} alt=''
+            />
+          </td>
+          <Row className='event-title'>{title}</Row>
           <Row className='event-date'>
-            {clockSymbol()}{'  '}
-            {getDateWithSlashes(eventDate.slice(0, 10))} {startTime} - {endTime}
+            {clockSymbol()}
+            {'  '}
+            {getDateWithSlashes(eventDate.slice(0, 10))} {startTime} -
+            {' '}
+            {endTime}
```

Comment:
<img width="749" alt="Screen Shot 2020-08-23 at 9 02 36 PM" src="https://user-images.githubusercontent.com/50391828/91002624-03c66980-e584-11ea-819c-d3374cb7a7d1.png">
get rid of the red `selected` outline